db/migrate/20170222131211_change_pool_columns_to_dates.rb

```diff
@@ -1,14 +1,18 @@
 class ChangePoolColumnsToDates < ActiveRecord::Migration
   def up
-    change_column(:katello_pools, :start_date, 'timestamp USING CAST(start_date AS timestamp without time zone)')
-    change_column(:katello_pools, :end_date, 'timestamp USING CAST(end_date AS timestamp without time zone)')
+    if connection.adapter_name.downcase != 'sqlite'
```

Comment:
I think it should be `sqlite3` based on https://github.com/theforeman/foreman/blob/develop/config/database.yml.example#L4